confd/service/loader_test.go

```diff
@@ -35,7 +33,7 @@ func TestConvertTotServicesShouldNotFailOnError(t *testing.T) {
 }
 
 func TestConvertToService(t *testing.T) {
-	registry := &mocks.Registry{}
+	registry := newMockConfigRegistry(t)
```

Comment:
Me: Ah yes, the constructor that binds a mock assertion to the test clean up. 
_scrolls further_
..
:cricket: :cricket: 
..
:thinking: Shouldn't be the manual mock assertion be deleted then?
..
:cricket: :cricket: 
..
_scrolls back, opens old code_
me, realizing the previous mocks were not asserted
![i-guess-memes](https://github.com/user-attachments/assets/010bb649-5fea-48ce-a45e-14ae1e779eab)


Anyhow good work in seeing that :muscle: